src/KalTileView.m

```diff
@@ -92,7 +92,7 @@ - (void)drawRect:(CGRect)rect
   
   NSUInteger n = [self.date day];
   NSString *dayText = [NSString stringWithFormat:@"%lu", (unsigned long)n];
-  CGSize textSize = [dayText sizeWithFont:font];
+  CGSize textSize = [dayText sizeWithAttributes:@{ NSFontAttributeName: font }];
```

Comment:
This produces slightly different results, but it gets round a couple lines below anyway, so we should be fine.

![2017-04-21_1805](https://cloud.githubusercontent.com/assets/217368/25301838/c945b884-26e5-11e7-9a75-621deaf2c334.png)